backend/apps/matching/configuration/participant.py

```diff
@@ -0,0 +1,24 @@
+from itertools import chain
+
+
+class ParticipantConfig:
```

Comment:
Only getters no setters?

Reply:
no, all variable setting is done in the initialisation and then the variables are `final` in a java sense, so there is no need for setters.